circle/circle.go

```diff
@@ -18,6 +18,15 @@ Square: 201.06192982974676
 func main() {
 	fmt.Println("The program prints the perimeter and the square of a circle given the radius.")
 	// ++ Your code here! ++
```

Comment:
optional: Feel free to remote this comment. We put it as some kind of a placeholder, but you don't really need them.